app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala

```diff
@@ -34,6 +34,9 @@ class AppConfig @Inject() (config: Configuration, contactFrontendConfig: Contact
   val xiEoriAddingDisabled: Boolean =
     config.getOptional[Boolean]("features.xi-eori-adding-disabled").getOrElse(false)
 
+  val submitAReportEnabled: Boolean =
```

Comment:
You are always able to submit a report. Can you change the flag name back to what we agreed please.